invenio_cli/helpers/commands.py

```diff
@@ -151,8 +151,25 @@ def _ensure_containers_running(self):
         docker_helper.start_containers()
         # TODO: Find faster way to procede when containers are ready
         time.sleep(30)  # Give time to the containers to start properly
+    
+    def destroy(self):
+        """Destroys env, containers."""
+        command = [
+            'pipenv', '--rm'
+            ]
+        subprocess.run(command, check=True)
```

Comment:
This could be compressed in one line. Usually we divide it when is a large command, in this case is quite straight forward. It's mostly styling and me being OCD tho :)
```suggestion
        subprocess.run(['pipenv', '--rm'], check=True)
```

or if you want to be explicit about the command:

```suggestion
        command = ['pipenv', '--rm']
        subprocess.run(command, check=True)
```